app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Great job so far. please, like the previous reviewer noted, Kindly remember, for the application to ask for confirmation of email Ensure` :confirmable` is enabled in the user model as well. Kindly set that up. 
You should see something similar to what is below:

		devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable



Don't forget to activate it as well in the devise migration file